README.md

```diff
@@ -67,6 +72,26 @@ wget https://github.com/kelseyhightower/confd/releases/download/v0.16.0/confd-0.
 chmod +x /usr/local/bin/confd
 ```
 
+##### Basic installation instruction from sources
```

Comment:
Could we suggest when build from the source is required?

Reply:
We can't suggest it because that is a user preference. I prefer sources because I don't like prebuild and like to install with `make install`. Our entire DevOps uses prebuild and put that binary somewhere in PATH.